ios/Classes/PTFlutterDocumentController.m

```diff
@@ -260,7 +273,7 @@ -(void)toolManager:(PTToolManager*)toolManager willRemoveAnnotation:(nonnull PTA
         [self.plugin documentController:self annotationsChangedWithActionString:annotationsWithActionString];
     }
     
-    NSString* xfdf = [self generateXfdfCommandWithAdded:Nil modified:Nil removed:@[annotation]];
+    NSString* xfdf = self.isAnnotationManagerEnabled ? [self.pdfViewCtrl.externalAnnotManager GetLastXFDF] : [self generateXfdfCommandWithAdded:Nil modified:Nil removed:@[annotation]];
```

Comment:
Isn't "will remove" too early to `GetLastXFDF`? Have you tested?

Reply:
I checked and you are right, calling `GetLastXFDF` there returns an incorrect XFDF. I have changed the implementation.